core/src/main/java/com/xatkit/core/recognition/RecognitionMonitorInflux.java

```diff
@@ -0,0 +1,600 @@
+        JsonArray sessionRecords = new JsonArray();
+        sessionObject.add("entries", sessionRecords);
+        int unmatchedCount = 0;
+        int matchedCount = 0;
+        double accConfidence  = 0.0;
+        for (FluxRecord sessionEntry : sessionData) {
+            JsonObject entryObject = getIntentData(sessionEntry);
+            sessionRecords.add(entryObject);
+            if (String.valueOf(sessionEntry.getValueByKey("is_Matched")).equals("false")) {
+                unmatchedCount++;
+            } else {
+                accConfidence += Double.parseDouble(
+                                    String.valueOf(
+                                        sessionEntry.getValueByKey("confidence")
+                                    )
+                                );
+                matchedCount++;
+            }
+        }
+        sessionObject.add("matchedUtteranceCount", new JsonPrimitive(matchedCount));
+        sessionObject.add("unmatchedUtteranceCount", new JsonPrimitive(unmatchedCount));
+        if (matchedCount > 0) {
+            sessionObject.add("avgSessionConfidence",
+                    new JsonPrimitive(accConfidence / (double) matchedCount));
+        }
+        return sessionObject;
+    }
+
+    /**
+     * Reads Record's Intent/utterance data and parses it as a JsonObject to be returned by the API.
+     * @param record
+     * @return JsonObject with record's data 
+     */
+    private JsonObject getIntentData(FluxRecord record){
+        String aux = String.valueOf(record.getValueByKey("_time"));
+        long time = Instant.parse(aux).toEpochMilli();
+        JsonObject obj = new JsonObject();
+        // Adding "generic" data
+        obj.addProperty("bot_id", String.valueOf(record.getValueByKey("bot_id")));
+        obj.addProperty("origin", String.valueOf(record.getValueByKey("origin")));
+        obj.addProperty("platform", String.valueOf(record.getValueByKey("platform")));
+        // Not sure if this is related to the intents or not for the session, but I am putting it just before the session data
+        obj.addProperty("parameters", String.valueOf(record.getValueByKey("matched_params")));
+        // Adding utterance/intent specific data 
+        obj.addProperty("session_id", String.valueOf(record.getValueByKey("session_id")));
+        obj.addProperty("timestamp", time);
+        obj.addProperty("utterance", String.valueOf(record.getValueByKey("utterance")));
+        obj.addProperty("is_matched", String.valueOf(record.getValueByKey("is_Matched")));
+        obj.addProperty("intent", String.valueOf(record.getValueByKey("matched_intent")));
+        obj.addProperty("confidence", Double.parseDouble(String.valueOf(record.getValueByKey("confidence")))); //confidence ha de ser un double al JSON!
+        return obj;
+    }
+
+
+    /**
+     * Closes connection to database. Changes should be commited, but check influxDB doc in case some actions need to be performed!
```

Comment:
No need to reference the InfluxDB documentation in the Javadoc.
Either it's an issue and it should be reported in Github, or there is something to keep in mind and it should be a comment inside the method (so it doesn't appear in the built Javadoc).
In general I'd avoid generic sentences like this: the InfluxDB documentation is probably pretty clear on this.

Reply:
I'll take care of this